app/controllers/addons_controller.rb

```diff
@@ -0,0 +1,51 @@
+class AddonsController < ApplicationController
+  def index
+    @addons = current_employee.restaurant.addons
+  end
+
+  def show
+    @addon = Addon.find(params[:id])

```

Comment:
`@addon = Addon.find(params[:id])`
This line is repeating.

- Create a set_addon method in private
- Add before_action callback to this method on required action.

Reply:
Do it for all other controllers too.

Reply:
Done